src/prop-of.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const { flip, prop } = require('ramda');
+
+/**
+ * Returns a function that when supplied a property returns the value of this, if it exists.
+ *
+ * @example
+ *   propOf({ name: 'Alexander' }, 'name') //=> 'Alexander'
+ *   propOf({ name: 'Alexander' }, 'age') //=> undefined
+ * @function
+ * @see https://ramdajs.com/docs/#flip
+ * @see https://ramdajs.com/docs/#prop
+ * @param {Object} obj The object to query
+ * @param {String} p The property name
```

Comment:
```suggestion
 * @param {String} propName The property name
```

Let's use `propName` to [match existing jsdocs](https://github.com/flybondi/ramda-land/blob/develop/src/number-prop.js#L30).